core/common/assets/js/api/core/hooks/base.js

```diff
@@ -364,7 +364,7 @@ export default class HooksBase extends elementorModules.Module {
 	 *
 	 * @throw {Error}
 	 */
-	runCallback( event, callback, args, result ) {
+	runCallback( event, callback, args, result ) { // eslint-disable-line no-unused-vars
```

Comment:
We need to consider all these occurrences

Reply:
Ok.